README.md

```diff
@@ -0,0 +1,28 @@
+# Udacity Intro to PyTorch Challenge
+
+Final challenge for the [PyTorch Scholarship Challenge from Facebook](https://www.udacity.com/facebook-pytorch-scholarship). The challenge is to create a PyTorch image classifier from scratch that will identify different species of flowers.
+
+![Scholarship Badge](https://github.com/coreen/pytorch_challenge/blob/master/ScholarshipBadge.png)
+
+## Dataset
+
+The provided dataset contains both a training and validation set. The zipped archive can be downloaded from https://s3.amazonaws.com/content.udacity-data.com/courses/nd188/flower_data.zip
+
+## Steps to run
+
+1. Download dataset
```

Comment:
Ideally this should specify where the dataset is downloaded within the folder and that it should be unzipped before using